Orange/tree.py

```diff
@@ -228,6 +228,11 @@ def get_instances(self, nodes):
         if subsets:
             return self.instances[np.unique(np.hstack(subsets))]
 
+    def get_indices(self, nodes):
```

Comment:
`get_instances` could call this function. Don't forget to handle the case when `get_indices` returns `None`.